alf/examples/main.py

```diff
@@ -92,26 +91,26 @@ def train_eval(train_dir, algorithm_ctor, evaluate=True,
             "Algorithm must be one of `OffPolicyAlgorithm`,"
             " `OnPolicyAlgorithm`. Received:", type(algorithm))
     trainer(
-        train_dir,
+        root_dir,
         env,
         algorithm,
         eval_env=eval_env,
         debug_summaries=debug_summaries)
 
 
 @gin.configurable
-def play(train_dir, algorithm_ctor):
+def play(root_dir, algorithm_ctor):
     """Play using the latest checkpoint under `train_dir`.
 
     Args:
-        train_dir (str): directory where checkpoints stores
+        root_dir (str): directory where checkpoints stores
         algorithm_ctor (Callable): callable that create an algorithm
             parameter value is bind with `__main__.train_eval.algorithm_ctor`,
             just config `__main__.train_eval.algorithm_ctor` when using with gin configuration
     """
     env = create_environment(num_parallel_environments=1)
     algorithm = algorithm_ctor(env)
-    on_policy_trainer.play(train_dir, env, algorithm)
+    on_policy_trainer.play(root_dir, env, algorithm)
```

Comment:
need to update on_policy_trainer.play to accept root_dir instead of train_dir

Reply:
updated